troubleshooting/cms_barebones_image_boot.md

```diff
@@ -84,11 +84,15 @@ cray.barebones-boot-test: DEBUG    Creating bos session template with etag:bc390
 The following manual steps may be performed to reproduce the actions of this script. The result should
 be the same as running the script.
 
-1. [Locate CSM Barebones Image in IMS](#csm-boot-steps-locate-barebones-image-in-ims)
-1. [Create a BOS Session Template for the CSM Barebones Image](#csm-boot-steps-bos-session-template)
-1. [Find an available compute node](#csm-boot-steps-node)
-1. [Reboot the node using a BOS session template](#csm-boot-steps-reboot)
-1. [Watch Boot on Console](#csm-boot-steps-watch-boot)
+  - [1. Steps the Script Performs](#1-steps-the-script-performs)
```

Comment:
The anchor names in this list also do not match the anchor names in the document

Reply:
I pulled back in the content that was there before. I didn't see that VSCode had automatically changed these values when I saved the file.